administrator/components/com_csp/src/Helper/ReporterHelper.php

```diff
@@ -76,4 +76,62 @@ public static function getCspTrashStatus()
 
 		return boolval($result);
 	}
+
+	/**
+	 * Check whether there are unsafe-inline rules published
+	 *
+	 * @return  boolean  Whether there are unsafe-inline rules published
+	 *
+	 * @since   4.0.0
+	 */
+	public static function getCspUnsafeInlineStatus()
+	{
+		$db = Factory::getDbo();
+		$query = $db->getQuery(true)
+			->select('COUNT(*)')
+			->from($db->quoteName('#__csp'))
+			->where($db->quoteName('blocked_uri'). ' = ' . $db->quote("'unsafe-inline'"))
```

Comment:
```suggestion
			->where($db->quoteName('blocked_uri') . ' = ' . $db->quote("'unsafe-inline'"))
```
Drone PHPCS: "Concat operator must be surrounded by a single space."